clayui.com/content/blog/2019-10-24-introducing-clay-v3.md

```diff
@@ -0,0 +1,130 @@
+| [Multi Step Nav](/docs/components/forms/multi-step-nav.html) | `@clayui/multi-step-nav` |
+| [Breadcrumb](/docs/components/navigation/breadcrumbs.html) | `@clayui/breadcrumb` |
+| [Form](/docs/components/forms/form.html), [Input](/docs/components/forms/input.html) | `@clayui/form` |
+| [Vertical Nav, Nav](/docs/components/navigation/vertical-navigation.html) | `@clayui/nav` |
+| [Panel](/docs/components/Panel.html) | `@clayui/panel` |
+| [Popover](/docs/components/popover.html) | `@clayui/popover` |
+| [Tabs](/docs/components/tabs.html) | `@clayui/tabs` |
+| Slider | `@clayui/slider` |
+
+#### Principles
+
+With the framework change that impacts the way we use and think about the use of components, we developed these new components following React's [composition](https://reactjs.org/docs/composition-vs-inheritance.html) approach, you can read more about [Clay's fundamentals of composition in our documentation](/docs/foundations/composing.html) if you want to dig deeper.
+
+### Deprecation warning for Clay v2
+
+Since we are entering v2 and a major framework change for our components, we are starting to deprecated v2 components. This means that we will not be working on new features for v2 but will still continue to support bugs and security issues. We will detail in other posts about how the gradual depreciation of v2 will work.
+
+### Release Process
+
+We are adopting package independent release, releasing packages with their independent version, this will make it easier to make gradual and specific releases of each component. We will test how this release process will work.
+
+#### SemVer
+
+We are adopting the [SemVer](https://semver.org/) versioning system to make it easy for you to upgrade our packages with confidence.
+
+```shell
+MAJOR.MINOR.PATCH
+```
+
+So you can see more often releases of `patch` and consequently` major` but that does not mean there will be big changes like v3 introducing a new rewrite. We hope these releases will be more continuous and help us iterator faster.
+
+### Migrate Guide
+
+We have prepared a migration guide from v2 to v3, keep in mind that v3 has had major changes this guide helps you come from Metal.js to React.js. We hope this can ease the pain for those unfamiliar with React.js.
+
+<div class="clay-ullist clay-ullist-margin-sm">
+
+- [Migrate the Clay components from v2 to v3](/docs/migration/migrate-the-clay-components-from-v2-to-v3.html)
+
+</div>
+
+### 3.x Roadmap
+
+We are working on developer experience enhancements and delivering new features to accelerate development. We have a few things for our roadmap.
+
+#### Data Display
+
+To help teams create complex and extensible data displays, Clay offers different Table components that can be used and configured on a high level.
+
+We will collect use cases and possible future data display cases, we will work on creating documentation and examples to help developers effectively display data across different applications.
```

Comment:
```suggestion
To help teams create complex and extensible data displays, Clay offers different Table components that can be used and configured on a high level. However, we are collecting current and future use cases so that we can create documentation and examples to help developers effectively display data across different applications. As we discover common patterns and uses, we will further evaluate the best way to provide components for users.
```